util/utils.hpp

```diff
@@ -7,9 +7,9 @@
 
```

Comment:
Can you remove this include? It breaks the build on github now that you added a new lib:
https://github.com/ratschlab/Project2020-seq-tensor-sketching/runs/1696839694?check_suite_focus=true

Reply:
Ehh, extra includes shouldn't break builds :thinking:  but yeah, gone now.

Reply:
Why not? If you include a file you don't depend on, the break should build :)